#goal definition
float32 accuracy
float32 distance
geometry_msgs/Point desired_point
---
#result definition
---
#feedback
float32 progress